examples/nodejs/src/snippets/utils/utils.ts

```diff
@@ -0,0 +1,58 @@
+import * as ScanbotSDK from "scanbotsdk";
+
+export async function extractImages(
+  filePath: string
+): Promise<ScanbotSDK.PageExtractionResult> {
+  return await ScanbotSDK.autorelease(async () => {
```

Comment:
I would say that if the method returns image, then it's a bit unnatural to have autorelease here. as it reuires the caller to do the operations inside the autorelease pool as otherwise all the images are cleared on exit